### PR TITLE
Created directory models. Added user.js, post.js, data.js to models. …

### PR DESCRIPTION
**Pull request by mrogers23458**
…Updated models with properties required in ticket.

### CODE REVIEW

File: moldels/data.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  {
+    sequelize,
+    timestamps: false,
+    freezeTableName: true,
+    underscored: true,
+    modelName: "data",
+  }
+);
+
+module.exports = Data;
```
Comment (Myuze) [inline moldels/data.js:40]:
```suggestion
module.exports = Data;

```

File: moldels/index.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    foreignKey: "data_id",
+    onDelete: "CASCADE"
+});
+
+Data.belongsTo(User, {
+    foreignKey: "data_id",
+    onDelete: "CASCADE"
+})
+
+  module.exports = { User, Post, Data };
```
Comment (Myuze) [inline moldels/index.js:24]:
```suggestion
  module.exports = { User, Post, Data };
  
```

**Review by Myuze — CHANGES_REQUESTED** [review]
Directory name should be models. Also, Capitalize the Model files as they are classes.

**Review by mrogers23458 — COMMENTED** [review]
pushed up new changes to use first letter capital on Models, and I updated spelling on moldels folder to models.

**Review by LordLowdar — COMMENTED** [review]
💩

**Review by ENguyen25 — APPROVED** [review]

**Review by LordLowdar — APPROVED** [review]

**Review by LordLowdar — APPROVED** [review]

**Review by LordLowdar — APPROVED** [review]

**Review by LordLowdar — APPROVED** [review]

**Review by LordLowdar — APPROVED** [review]
Sounds good

**Review by LordLowdar — APPROVED** [review]
💥

**Review by LordLowdar — APPROVED** [review]
💥

**Pull request merged**